src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -1007,7 +1010,7 @@ public final Response cancelReservations(@Context final HttpServletRequest reque
             || bookingManager.isReservationMadeByRequestingUser(userLoggedIn, userOwningBooking, event)
             && userAssociationManager.hasPermission(userLoggedIn, userOwningBooking)) {
           if (bookingManager.hasBookingWithAnyOfStatuses(eventId, userId, new HashSet<>(Arrays.asList(
-              BookingStatus.CONFIRMED, BookingStatus.WAITING_LIST, BookingStatus.RESERVED)))) {
+              CONFIRMED, BookingStatus.WAITING_LIST, BookingStatus.RESERVED)))) {
```

Comment:
something weird going on here - `CONFIRMED` is unqualified, but the others have a class?

Reply:
Ah I think that's just because I directly imported CONFIRMED

`import static uk.ac.cam.cl.dtg.isaac.dos.eventbookings.BookingStatus.CONFIRMED;`

I'll change that to just use BookingStatus.CONFIRMED for consistency